apps/server/src/shared/infra/rabbitmq/exchange/files-storage.ts

```diff
@@ -32,7 +32,7 @@ export interface ICopyFilesOfParentParams {
 export interface IFileRecordParams {
 	schoolId: EntityId;
 	parentId: EntityId;
-	parentType: FileRecordParentType;
+	parentType: FileRecordParent;
```

Comment:
Why did you change the name? I would say FileRecordParentType was the correct name. FileRecordParent sounds like an actual object to me. Also in this line you name the prop parentType and also in other files.

Reply:
Because FileRecordParent is an enum and not a type. So type is wrong in this place. Suffixing it with enum instead isn't needed because our IDE is allready telling us that it is an enum.

Reply:
I believe the convention is to name an enum after the thing it enumerates. And this one enumerates a type. There are also at least 10 similar enums in our code that are called "...Type". So even if it is not a type definition in the typescript sense, we seem to have the convention to call this kind of thing "type" (which is okay in my opinion). If it wouldn't be okay to name this type, you would also have to rename the parentType prop because it doesn't store a type in the strong sense either.

Reply:
We share the understanding that an enum should be named after the thing it enumerates. And from my point of view it enumerates the File Record Parent. Or isn't it? Why do I need the type suffix? And on top of that, I think that it's not good to use a word that has a fixed technical meaning in another sense.

On the other hand you are totally right that there are a lot of enums that have the type suffix. But there are also quite a few that don't have type at the end. @CeEv What do you think? How should we proceed here?

Reply:
I would say the FileRecordParent is the actual object, e.g. a task. While the enum states its type, e.g. that the parent is of type Task.

I've been thinking again and I believe this is also correct in a narrow sense in TypeScript because Task is a class and a class defines a type. At first I was thinking about type aliases (https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-aliases) where the keyword "type" is actually used, but those are not the only types in TypeScript.

Reply:
We keep the old naming FileRecordParentType because naming of enums and types is not consistent at all